nodejs/eks/cluster.ts

```diff
@@ -260,15 +311,38 @@ export class Cluster extends pulumi.ComponentResource {
         }, { parent: this });
 
         // Enable access to the EKS cluster for worker nodes.
+        const instanceRoleMapping: RoleMapping = {
+            roleArn: instanceRoleARN,
+            username: "system:node:{{EC2PrivateDNSName}}",
+            groups: ["system:bootstrappers", "system:nodes"],
+        };
+        const roleMappings = pulumi.all([pulumi.output(args.roleMappings || []), instanceRoleMapping])
+            .apply(([mappings, instanceMapping]) => {
+                return jsyaml.safeDump([...mappings, instanceMapping].map(m => ({
+                   rolearn: m.roleArn,
```

Comment:
Assuming the `rolearn` casing is correct, is there a reason to not just make `RoleMapping` use the same casing, and doing something like `jsyaml.safeDump([...mappings, instanceMapping])`?

Reply:
The `rolearn` casing is correct, yes. I opted to make the user-facing API as idiomatic as possible.